demo/long_video_demo.py

```diff
@@ -88,6 +95,11 @@ def show_results():
 
         ret, scores = inference()
 
+        if ret and stride > 0:
+            pred_stride = int(sample_length * stride)
+            for i in range(pred_stride):
+                frame_queue.popleft()
```

Comment:
Got! I think it is better to move this to `def inference` to avoid repeat popleft 

```python
def inference():
    if len(frame_queue) != sample_length:
        # Do no inference when there is no enough frames
        return False, None

    cur_windows = list(np.array(frame_queue))
    img = frame_queue.popleft()

    if stride >0:
        pred_stride = max(1, int(sample_length * stride))
        for _ in range(pred_stride - 1):
            frame_queue.popleft()

    ...
```
Better to rerun for checking.

Reply:
OK. Besides, I would like to remove global args.